core/block_dragger.js

```diff
@@ -10,61 +10,61 @@
  */
 'use strict';
 
-goog.provide('Blockly.BlockDragger');
-
-goog.require('Blockly.blockAnimations');
+goog.module('Blockly.BlockDragger');
+goog.module.declareLegacyNamespace();
+
+const BlockSvg = goog.requireType('Blockly.BlockSvg');
+const Coordinate = goog.require('Blockly.utils.Coordinate');
+const IBlockDragger = goog.require('Blockly.IBlockDragger');
+const IDragTarget = goog.requireType('Blockly.IDragTarget');
+const InsertionMarkerManager = goog.require('Blockly.InsertionMarkerManager');
+const WorkspaceSvg = goog.requireType('Blockly.WorkspaceSvg');
+const {BLOCK_DRAG, BLOCK_MOVE, fire, get, getGroup, setGroup} = goog.require('Blockly.Events');
+const {DEFAULT, Type, register} = goog.require('Blockly.registry');
+const {disconnectUiEffect, disconnectUiStop} = goog.require('Blockly.blockAnimations');
+const {startTextWidthCache, stopTextWidthCache} = goog.require('Blockly.utils.dom');
```

Comment:
I know this is probably inconvenient to update, but I think the requires for `Blockly.Events`, `Blockly.registry`, and `Blockly.utils.dom` would be better not destructured because without the namespace the methods and constants aren't as clear.

(`Blockly.blockAnimations` I think is fine destructured)

Reply:
done